pkg/controller/vsphere/machine_scope.go

```diff
@@ -105,12 +105,16 @@ func (s *machineScope) PatchMachine() error {
 	}
 	s.machine.Status.ProviderStatus = providerStatus
 
+	statusCopy := *s.machine.Status.DeepCopy()
```

Comment:
can we have a unit for this as we do for other actuators?

Reply:
Can this go in a follow-up PR?

Reply:
I think that can be a follow up as we have done in previous cases